apps/web-mzima-client/src/app/settings/users/users.component.scss

```diff
@@ -38,6 +38,18 @@
         }
       }
     }
+    //     .p-datatable .p-sortable-column.p-highlight{
+    // color: var(--color-light-contrast);
+    //     }
+    // .p-datatable .p-sortable-column.p-highlight .p-sortable-column-icon{
+    //   color: var(--color-light-contrast);
+    // }
+    .p-datatable .p-datatable-thead > tr > th {
+      color: var(--color-light-contrast);
+    }
+    .p-datatable .p-sortable-column .p-sortable-column-icon {
+      color: var(--color-light-contrast);
+    }
```

Comment:
Since this is an `scss` setting, let's use `scss` method instead of the regular `css` method of accessing children elements @kivuvarosekivuvan

Reply:
@Ifycode You mean accessing children using the ampersand operator(&)?